src/contributing/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Contributing to the dxw Accessibility Manual
+navigation_order: 0
+---
+
+The dxw Accessibility Manual is public and anyone, inside or outside of dxw, can suggest changes to it. This means you!
+
+If you know how to suggest changes using GitHub, the repository is available at https://github.com/dxw/accessibility-manual
+
+If you don't know how to use GitHub, you can suggest changes using our content management system, Netlify CMS, which does things on GitHub for you.
+
+## What you'll need to suggest a change
+
+You will need a GitHub account. If you don’t already have one, then sign up at https://github.com/
+
+You do not need to be added to the dxw GitHub organisation to be able to suggest changes.
+
+## Suggesting a change to an existing page
+
+1. On the page you want to edit, select the yellow ‘Edit this page’ link.
+
+Or if you prefer to use the admin backend directly, choose the page you want to change from the backend under 'Collections'.
+
+2. Log in with your GitHub account
+
+3. Select ‘fork the repo’ if you are asked whether you want to fork the repo
+
+4. Make your edits in the ‘Rich Text’ editor
+
+5. Select ‘Save’ at the top of the page
+
+6. Select ‘Status: Draft’ and choose ‘In review’
```

Comment:
Should these have full stops like the others? (I think generally there's a bit of inconsistency on full stops 😱)

Reply:
I can come back to this but thank you for pointing it out :)